src/app/docs/action-hub/action-hub-docs.component.html

```diff
@@ -109,19 +109,17 @@ <h2 class="sky-font-heading-2">Additional Content</h2>
       </sky-docs-design-guideline>
 
       <sky-docs-design-guideline heading="Content area">
-        <p>Content should follow one or more of these principles:</p>
+        <p>Content follows one or more of these principles:</p>
         <ul>
           <li>Actionable (inside or outside the system)</li>
           <li>Makes the user better at their job</li>
-          <li>
-            Pre-analyzed or synthesized to communicate meaning and context
-          </li>
+          <li>Pre-analyzed or synthesized to communicate meaning and context</li>
```

Comment:
This project is using `prettier` via [@skyux-sdk/builder-code-formatter](https://github.com/blackbaud/skyux-sdk-builder-code-formatter) -- if you run `skyux format` within this project, this change gets undone.

The CI checks formatting and might balk at this.